### PR TITLE
#183031485-USERS SHOULD BE ABLE TO LOGIN

### PR DESCRIPTION
**Pull request by ChrisOnsando**
#### What does this Pull Request do?
- Implements the login feature for the user
- Create a login form
#### Description of Task to be completed?
- Users should be able to login
- Validates the user credentials, then login the user
#### How should this be manually tested?
- run `npm start`
- Navigate to the login page
- Input email and password in the form
- Submit to login
#### Any background context you want to provide?
- With correct email address and password, users should be able to login to the application
#### What are the relevant pivotal tracker story?
- (#183031485)- https://www.pivotaltracker.com/story/show/183031485
#### Screenshots (if appropriate)
Screenshot from 2022-08-30 12-47-38
Screenshot from 2022-08-26 08-32-24
#### Questions:
- N/A

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -29,4 +29,4 @@
         "react/prop-types": "off",
         "jsx-a11y/label-has-associated-control": "off"
     }
-}
+}
```
Comment (KaigaMunene) [inline .eslintrc.json]: space after this

File: src/assets/styles/pages/_login.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+login-form {
+  display: flex;
+}
```
Comment (KaigaMunene) [inline src/assets/styles/pages/_login.scss]: space after this

File: .eslintrc.json (hunk 1 above)
Comment (ChrisOnsando) [reply .eslintrc.json]: I'll add the spaces

File: src/assets/styles/pages/_login.scss (hunk 2 above)
Comment (ChrisOnsando) [reply src/assets/styles/pages/_login.scss]: Added

**Pull request merged**